virttest/utils_misc.py

```diff
@@ -2772,10 +2818,12 @@ def format_windows_disk(session, did, mountpoint=None, size=None, fstype="ntfs")
                 logging.info("Clear readonly bit on 'Disk%s'" % did)
                 session.cmd(set_rw_cmd)
 
-            if size is None:
+            if not size:
                 mkpart_cmd = 'echo create partition primary >> disk'
             else:
-                mkpart_cmd = 'echo create partition primary size=%s >> disk' % size
+                mkpart_cmd = 'echo create partition primary size=%s '
+                mkpart_cmd += '>> disk'
+                mkpart_cmd = mkpart_cmd % size
             mkpart_cmd = ' '.join([cmd_header, mkpart_cmd, cmd_footer])
```

Comment:
Why modify this? Seems no difference with previous one, which I prefer as it looks pithier